### PR TITLE
update to PVR API 5.7.0

### PR DESCRIPTION
**Pull request by fuzzard**
Stub IsEPGTagPlayable as not implemented
Stub IsEPGTagRecordable as not implemented
Stub GetEPGTagStreamProperties as not implemented

Dont merge until https://github.com/xbmc/xbmc/pull/12689 is merged into kodi master
Will track and update as/if ksooo modifies

### CODE REVIEW

File: src/client.cpp [hunk 1]
```diff
@@ -363,6 +363,9 @@ PVR_ERROR GetTimerTypes(PVR_TIMER_TYPE types[], int *size) { return PVR_ERROR_NO
 bool IsTimeshifting(void) { return false; }
 bool IsRealTimeStream(void) { return true; }
 PVR_ERROR SetEPGTimeFrame(int) { return PVR_ERROR_NOT_IMPLEMENTED; }
+PVR_ERROR IsEPGTagPlayable(const EPG_TAG* tag, bool* bIsPlayable) { return PVR_ERROR_NOT_IMPLEMENTED; }
+PVR_ERROR IsEPGTagRecordable(const EPG_TAG* tag, bool* bIsRecordable) { return PVR_ERROR_NOT_IMPLEMENTED; }
+PVR_ERROR GetEPGTagStreamProperties(const EPG_TAG* tag, PVR_NAMED_VALUE* properties, unsigned int* iPropertiesCount) { return PVR_ERROR_NOT_IMPLEMENTED; }
```
Comment (ksooo) [inline src/client.cpp]: You could omit the names of the vars as some compilers tend to complain about unused vars otherwise.

Comment (fuzzard) [inline src/client.cpp]: The rest are a mix of with and without. Ill add a commit to change all over to without.
Out of curiosity, what compilers complain? msvc?

Im only building on mac os using cmake, and monitoring the travis ci builds on linux/xcode.

File: depends/common/hdhomerun/hdhomerun.txt [hunk 2]
```diff
@@ -1 +1 @@
-hdhomerun https://github.com/Silicondust/libhdhomerun 5a22c0fde9c58279641d30372ddbdbdd48e0a300
+hdhomerun https://github.com/Silicondust/libhdhomerun 0d3b72c1ccf1502a73c7f413e7d3382d9ee732c6
```
Comment (zcsizmadia) [inline depends/common/hdhomerun/hdhomerun.txt]: Please remove this. I've created a seperate PR to change it to master. 

I do not know why the git tree becomes invalid, after SiliconDust checks something in ?!

Comment (fuzzard) [inline depends/common/hdhomerun/hdhomerun.txt]: Removed that commit. My experience with it was only the travis ci xcode tests were failing. I assume it was something to do with the checkout depth. Wasnt sure if you wanted to just go blanket master branch. The last few changes on the libhdhomerun branch have been fairly benign, but whos to say what will happen in the future.
Guess we just deal with that if any problems pop up down the line.

**Comment by ksooo** [comment]
Could you please squash down to a single commit? Then I'm happy to merge this.

**Comment by fuzzard** [comment]
Done. just fyi, the build failure will be because of the other PR needing to be merged for the travis xcode clients to suceed.

**Comment by ksooo** [comment]
> Done. just fyi, the build failure will be because of the other PR needing to be merged for the travis xcode clients to suceed.

 sure.

**Comment by zcsizmadia** [comment]
Thanks @fuzzard !

**Comment by zcsizmadia** [comment]
@ksoo I've change the libhdhomerun vaersion to the latest commit number. I had issue with the  libhdhomerun in the past, that if we are not using the master commit, the cmake engine fail with the libhdhomerun git clone, saying tree does not exist (which is not really true). I haven't debugged the cmake engine, but just looked at the cmake code, and it seems the commit number is not in the git log. As @fuzzard suggested, it might be because of the checkout depth is set to 1?

**Comment by ksooo** [comment]
no idea, sorry - I'm not at all familar with cmake.

**Pull request merged**